index.html

```diff
@@ -159,7 +154,6 @@ <h2>
       <pre class="idl">
         [SecureContext, Exposed=(Window)]
         interface ScreenFold : EventTarget {
```

Comment:
We should rename ScreenFold etc as well, maybe different PR

Reply:
Yes, part of the rename.